### DISCUSSION TITLE
p-inplace has no maximum length or minimum length property

### DISCUSSION THREAD

**Discussion by accuratesoftwaresolution** [primefaces/community#1419] [PrimeNG] [answered]
p-inplace has no option to limit the user input data. is there any solution for this ?

Can't bind to 'maxlength' since it isn't a known property of 'p-inplace'.
1. If 'p-inplace' is an Angular component and it has 'maxlength' input, then verify that it is part of this module.
2. If 'p-inplace' is a Web Component then add 'CUSTOM_ELEMENTS_SCHEMA' to the '@NgModule.schemas' of this component to suppress this message.
3. To allow any property add 'NO_ERRORS_SCHEMA' to the '@NgModule.schemas' of this component.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

p-inplace is not an input component, it uses angular content projection to display content, we cannot add maxlength attribute. If you're using input inside the p-inplace, you need to add it by yourself to your input, we cannot interfere this.
